pkg/generate/rds_test.go

```diff
@@ -403,51 +403,51 @@ func TestRDS_DBInstance(t *testing.T) {
 		ko.Status.DBParameterGroups = f14
 	}
 	if resp.DBInstance.DBSubnetGroup != nil {
-		f16 := &svcapitypes.DBSubnetGroup_SDK{}
+		f15 := &svcapitypes.DBSubnetGroup_SDK{}
```

Comment:
The numbers are shifted since the shape is not considered at all during the count, hence the diff here is no-op.